Vagrantfile

```diff
@@ -55,18 +55,18 @@ Vagrant.configure("2") do |config|
     ./build.sh
 
 # Install opam and dependencies for compiling xapi
-    sudo apt-get install -y opam m4 libxen-dev
-    opam init -a --compiler=4.04.2 -y
-    eval `opam config env`
+    sudo apt-get install -y m4 libxen-dev
+    # Install opam 2:
+    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
+    opam init --disable-sandboxing --auto-setup --compiler=4.06.1 --yes
```

Comment:
I think disabling sandboxing is required for containers (the sandboxing tool may not be able to launch), but should be fine for virtual machines. I have sandboxing enabled on my machine.

Reply:
I did it because there is no bubblewrap package for this older version of ubuntu, plus recently we found it impossible to install opam-depext with sandboxing enabled

Reply:
OK, lets keep it disabled. Would it be worth updating the version of Ubuntu to the latest 18.04 LTS?

Reply:
Yes, I wanted to do that, but I'm not sure how to. And I was afraid it would break many things.
But we should update it.